airflow/contrib/hooks/gcp_dataproc_hook.py

```diff
@@ -141,24 +141,99 @@ def build(self):
         return self.job
 
 
+class _DataProcOperation(LoggingMixin):
+    """
+    Continuously polls Dataproc Operation until it completes.
```

Comment:
This should be on the previous line, according to https://www.python.org/dev/peps/pep-0257/#one-line-docstrings — this is a one-line docstring. Similarly elsewhere for single-line docstrings.